packages/ckeditor5-html-support/src/generalhtmlsupportconfig.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module html-support/generalhtmlsupport
```

Comment:
```suggestion
 * @module html-support/generalhtmlsupportconfig
```

And check if there are any `{@link ...}` outside that points to `GeneralHtmlSupportConfig`.